### PR TITLE
Refactor AL code for ViT-L/32

### PR DESCRIPTION
**Pull request by y0ast**
This PR refactors the AL script to use ViT-L/32, and also fixes the config overwrites (by introducing its own experiment config) to make it easier to run internally.

@zi-w @dustinvtran

### CODE REVIEW

**Review by zi-w — APPROVED** [review]

**Review by dustinvtran — COMMENTED** [review]
As mentioned in the meeting, the config LGTM, with the only major difference being 1/2 the batch size.

Starting with the best learning rate from our finetuning jobs is OK. You may still need to tweak it when we optimize for best possible performance. For example, the rule of thumb of 1/2 batch_size = 2x learning rate means 2x our current best learning rate is also a reasonable place to start.
